### DISCUSSION TITLE
Adjust time remaining based on playback speed

### DISCUSSION THREAD

**Discussion by ktwingstrom** [LeoKlaus/plappa#221] [Feature Requests]
One feature I really liked from the Prologue app I used to use was when you put the book in say 1.5x speed the time remaining would adjust to show how much ACTUAL time was left based on the speed I was listening. That would be great to see and would probably be pretty easy to implement!

**Comment by LeoKlaus** [comment]
I added this with the 1.3 update a couple of months ago (#140), can you make sure you're on the latest version of plappa?

**Reply by ktwingstrom** [reply]
Ok hang on. So I rolled back to the latest official version and removed the beta. And it seems like it updates the time remaining on the current file but not the whole book. https://github.com/user-attachments/assets/1c4084e5-9919-41f8-9665-49508eb465ba

**Comment by kevin-twingstrom** [comment]
There must have been a bug with my install because it's working fine now!  I downloaded the latest Beta from testflight but before I was on the latest as well....interesting.  Thanks for responding, loving this app!

**Reply by LeoKlaus** [reply]
I just released an update to TestFlight today to address this :D

**Reply by kevin-twingstrom** [reply]
Oh got it haha!  Alright thanks so much!
